webapp/components/ContributionForm/index.vue

```diff
@@ -75,6 +107,13 @@ export default {
       disabled: false,
       slug: null,
       users: [],
+      dropzoneOptions: {
+        url: 'https://httpbin.org/post',
```

Comment:
As @Tirokk noted, this is a temporary solution. We should prioritize adding our own restful endpoint for the images to be uploaded to, and then store them in the backend upon form submit.

If there is a better temporary solution, I can change this